contracts/MarketItem.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import '@openzeppelin/contracts/utils/Counters.sol';
+import '@openzeppelin/contracts/token/ERC721/ERC721.sol';
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+contract MarketItem is ERC721, ERC721URIStorage, Ownable {
+    using Counters for Counters.Counter;
+
+    Counters.Counter private _tokenIdCounter;
+
+    constructor() ERC721('MarketItem', 'MTK') {}
+
+    function safeMint(address to, string memory uri) public onlyOwner returns (uint256) {
```

Comment:
@ymincheva Now only the owner can mint
Currenlty the owner is the deployer (your personal address), although the `Marketplace.sol` will be the account that will mint
So I beleive that `onlyOwner` should be removed except the `Marketplace.sol ` is the deployer